### ISSUE TITLE
Allow HUB to just proxy videos from nodes, without chaching.

### ISSUE THREAD

**Issue by vitrilo** [groupon/Selenium-Grid-Extras#301]
Current solution for Videos - its stop recording on Node and then in background upload it to HUB. As result - HUB may receive videos with significant lag - sometimes minutes on hi-load (or may be it just in my case?).
From my point of view, may be more practical to HUB will just show the links videos and on video file request will just proxy this call to specific Node. For this solution Hub should cache SessionId-NodeId map instead of Video files itself. 

Additionally, SessionId-NodeId map - it very useful to keeping for a long time on HUB for troubleshooting purposes (not sure it API support this currently). 

If this feature is considerable as reasonable - I can prototype it and fill PR.

**Comment by sanghviharshit** [comment]
Do the videos on nodes automatically get uploaded to the Hub? Or you have setup some script for that? I also need a solution similar to this and I was thinking if the hub can accept the video recording command for a session and forward it to the node automatically as hub is already aware of the sessions and map the request to correct node. It'd be great if the hub can also show the recorded videos from the node machines.

**Comment by vitrilo** [comment]
@sanghviharshit Yes, Selenium-Grid-Extras automatically correctly  uploaded videos from Nodes to Hub with some **lag** (from seconds to minutes depending on load). 
At first my workarond was a script to wait some time (two minutes) after session ends, end try download from Hub,
But my current workarond - to script try download directly from all Nodes (video should in one of it, all other returns 404) :) . The positive of such workarond - video are available immediately.

**Comment by sanghviharshit** [comment]
That is strange. I have just started setting up selenium grid and I don't see any videos from the node after running the tests. I see them when I go to <node-ip>:3000, but I don't see any nodes or their videos on the <hub-ip>:3000 page. I also checked the video_output directory on my hub, but I don't see anything there. Is there any way to debug what might be wrong with my setup?

P.S. sorry to hijack your thread 😢

**Comment by vitrilo** [comment]
@sanghviharshit Logs should help. 
FYI: Recheck that your tests should use HUBs url, if you point a tests directly to some node (like my mistake) - HUB will dont know anything about this, so no video will be copied to HUB.

**Comment by sanghviharshit** [comment]
I am using the correct hub URL, but I will dig through the logs why videos are not showing up. Thank you!
